index.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <link rel="stylesheet" href="styles.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;300&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@100&display=swap" rel="stylesheet">
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Marwan Portfolio</title>
+  </head>
+  <body>
+    <header>
+      <nav class="navbar">
+        <div class="name-container">
+          <a href="index.html">
+            <h2>Marwan</h2>
+          </a>
+        </div>
+        <div>
+          <div class="bar1"></div>
+          <div class="bar2"></div>
+          <div class="bar3"></div>
+        </div>
+      </header>
+    </div>
+    <div class="links-container">
+      <ul>
+        <li>
+          <a>Portfolio</a>
+        </li>
+        <li>
+          <a>About</a>
+        </li>
+        <li>
+          <a>Contact</a>
+        </li>
+        <li>
+          <a
+          ><img
+          src="assets/Normal Button/Tertiary/Disabled.png"
+          alt="Message"
+          /></a>
+        </li>
+      </ul>
+    </div>
+  </nav>
+  
+    <div class="header">
+      <div class="welcome">
+        <h1>
+          Hey There. I<span>&#39;</span>m Marwan
+        </h1>
+      </div>
+      <div class="job-desc">
+        <h2>I<span>&#39;</span>m a Software Developer</h2>
+      </div>
+      <div class="intro">
+        <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+          don<span>&#39;</span>t hestiate to contact me.</p>
+      </div>
```

Comment:
- [Optional] Consider fixing the font sizes of your texts and the spacing between your texts to match the given template. You are expected to stick to design as much as possible.
### Template
![image](https://user-images.githubusercontent.com/94551484/219365847-bcbdb6fe-f36a-4a43-afde-61b3dbfaf031.png)
### Your design
![image](https://user-images.githubusercontent.com/94551484/219365732-99bb4ecc-a829-49fa-804c-83bbff09e9e6.png)